Libraries/Opc.Ua.Client/Session.cs

```diff
@@ -4960,11 +4968,14 @@ private void OnPublishComplete(IAsyncResult result)
                     out acknowledgeResults,
                     out acknowledgeDiagnosticInfos);
 
+                LogLevel logLevel = LogLevel.Warning;
                 foreach (StatusCode code in acknowledgeResults)
                 {
-                    if (StatusCode.IsBad(code))
+                    if (StatusCode.IsBad(code) && code != StatusCodes.BadSequenceNumberUnknown)
                     {
```

Comment:
suppress this error when the server altready cleaned up the sequence number. Also print only once a Warning, then switch to trace.